packages/suite/src/views/settings/SettingsLoading.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { H3, Spinner, motionEasing } from '@trezor/components';
+import styled from 'styled-components';
+import { spacingsPx } from '@trezor/theme';
+import { Translation } from 'src/components/suite';
+import { motion, AnimatePresence } from 'framer-motion';
+
+const Container = styled(motion.div)`
+    background-color: ${({ theme }) => theme.backgroundTertiaryDefaultOnElevation0};
+    border-radius: 48px;
+    display: flex;
+    align-items: center;
+    margin-bottom: ${spacingsPx.md};
+    overflow: hidden;
+`;
+
+const SpinnerContainer = styled.div`
+    background-color: ${({ theme }) => theme.backgroundSurfaceElevation3};
+    border-radius: 50%;
+    margin: ${spacingsPx.sm};
+    padding: ${spacingsPx.md};
+`;
+const Content = styled.div`
+    margin: ${spacingsPx.sm};
+`;
+const Description = styled.div`
+    color: ${({ theme }) => theme.textSubdued};
+`;
+
+export const SettingsLoading = () => (
+    <>
+        <AnimatePresence initial={false}>
```

Comment:
since you've removed the 

```
 <AnimatePresence initial={false}>
            {isLoading && (
```

The animation stopped working :( In order for the `exit` animations in `framer-motion` to work, a `motion` element should be wrapped by `AnimatePresence` **and**, crucially, the animate presence should not be unmounted together with what it disappearing. I will fix the animation, as well as add an entrance animation since it's possible to observe this banner appearing in settings from now on.

Reply:
Done cf93aa919e017248e9fc9c8bb24bb5db2d4c2cd3